app/resources/numismatics/issues/numismatics/issue_change_set.rb

```diff
@@ -81,10 +81,10 @@ def primary_terms
           :shape,
           :color,
           :edge,
-          :object_date,
-          :earliest_date,
-          :latest_date,
-          :era,
+          [:earliest_date,
+           :latest_date],
+          [:era,
+           :object_date],
```

Comment:
Might look better (scan easier) if the nested arrays were formatted something like this:

```
          [
            :earliest_date,
            :latest_date
          ],
          [
            :era,
            :object_date
          ],
```